res/app/device-list/details/device-list-details-directive.js

```diff
@@ -277,6 +279,20 @@ module.exports = function DeviceListDetailsDirective(
         }
       })()
 
+      // check if new column needs a new scope or not
+      // and build accordingly
+      function buildColumn(col, device) {
```

Comment:
yeah, I am trying to get rid of angular stuff from device list. Thanks for the suggestions.